.github/workflows/scripts/run-tests.sh

```diff
@@ -1,30 +1,16 @@
 #!/bin/bash
 
-set -e
-
-LOGFILE="$GITHUB_WORKSPACE/backend/workingdir/log/globaleaks.log"
-ACCESSLOG="$GITHUB_WORKSPACE/backend/workingdir/log/access.log"
-
-function atexit {
-  if [[ -f $LOGFILE ]]; then
-    cat $LOGFILE
-  fi
-
-  if [[ -f $ACCESSLOG ]]; then
-    cat $ACCESSLOG
-  fi
-}
-
-trap atexit EXIT
+backend_test_failed=0
```

Comment:
This file has been edited across multiple commits with instances of code being removed and then re-added. To streamline the review, the offending commits should be removed, either by rebasing or squashing them into a single, meaningful commit.